src/data_access/FileDashDataAccessObject.java

```diff
@@ -51,6 +49,31 @@ public List<AccountInfo> getAccounts() {
         return this.accounts;
     }
 
+    /**
+     * Method that deletes an account with a unique title and username combination
+     *
+     * @param title    the title of the account to be deleted
+     * @param username the username of the account to be deleted
+     */
+    @Override
+    public void deleteAccount(String title, String username) {
```

Comment:
This method in the DAO is implemented correctly

Reply:
Thanks!